BLSQ/iaso
# Polio 1321 frontend button to download completeness data on campaigns page

Explain what problem this PR is resolving
- Change the csv export on campaigns
Related JIRA tickets : [POLIO-1321](https://bluesquare.atlassian.net/browse/POLIO-1321)

## Self proofreading checklist


## Changes
- Remove the xlsxSerializer
- Add a method to export campaign csv

## How to test
- Go on the campaign page
- Try to export csv
- Check the csv file content

## Print screen / video
[Screencast from 2023-12-26 10-09-15.webm](https://github.com/BLSQ/iaso/assets/19631540/1f1f9885-6d45-4d50-abb1-84522500e322)


[POLIO-1321]: https://bluesquare.atlassian.net/browse/POLIO-1321?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
